StortSpelProjekt/Game/src/main.cpp

```diff
@@ -609,6 +610,52 @@ Scene* ShopScene(SceneManager* sm)
     return scene;
 }
 
+void logSchtuff(const std::string& name)
```

Comment:
What is this?

Reply:
Something that should not exist, thank you for spotting it 👍